recipes/ohpipeline/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  "1.139.1000":
+    source:
+      url: "https://github.com/openhome/ohPipeline/archive/refs/tags/ohMediaPlayer_1.139.1000.tar.gz"
+      sha256: "070eac6d974bbe520d0b292df1ca4a72160a2b607a60dabdaa0971c3ef96e5e4"
+    cmake:
+      url: "https://raw.githubusercontent.com/merakiacoustic/ohPipeline/ohMediaPlayer_1.139.1000_meraki2/CMakeLists.txt"
+      sha256: "f9e11d4adb8426a8375dae02e3b641d66efc6ef3abd6f96cb81f6e85352083a3"
```

Comment:
I know I might be a bit late for the review but think we should not incorporate new build systems to a 3rd party library as there might be errors or subtle behavior changes for the end user. However, I understand the effort here and the usage of an external fork-provided CMakeLists. For future iterations of this recipe, let's try to use the original waf build system and stick to only the sources and instructions provided by the library. Thanks!

Reply:
Making the original work with Conan probably requires more code than just using CMake but ends up being less flexible...

Reply:
True. But it would be nightmare mostly for me, either way ;) The current version of the library doesn't even compile unless you have non-free SDK, so I'll need to patch it to handle this as an option.